code-challenges.test.js

```diff
@@ -25,11 +27,27 @@ var people = [
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is a president of the galaxy.", "Arthur Dent is a radio employee."]
 
-
+describe("sentence", () => {
+  it("returns an array with a sentence about each person with their names capitalized", () => {
+    expect(sentence(person)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is a president of the galaxy.", "Arthur Dent is a radio employee."])
+    
+  })
+})
 
 // b) Create the function that makes the test pass.
-
-
+// create a function called sentence that takes in an array
+const sentence = (array) => {
+  // create a variable with an empty array that will store each object
+  let allSentence = []
+  array.forEach(person => {
+    `${person.name} is a ${person.occupation}.`
+    allSentence.push(array)
+  })
+  return allSentence
+}
+console.log(sentence())
+
+// I know the forEach method should be used but i was not able to find the right way to implement it in my code. I would run into errors such as undefined and [ [Function: sentence], [Function: sentence], [Function: sentence] ].
```

Comment:
Yeah this is close, I hope going over the problem in class was helpful for thinking about how to manipulate the data within objects. You were on a right track with the forEach and I know those errors can be frustrating.